1-Vanilla/styles/style.css

```diff
@@ -0,0 +1,167 @@
+:root {
+    --logo-blue: #0b79a5;
+    --background-blue: #e5f6fd;
+    --hover-blue: #d7f2fd;
+    --shadow-blue: #085b7c;
+    --dark-green: rgb(48, 78, 58);
+    --light-green: rgb(150, 202, 167);
+    --dark-red: rgb(78, 25, 23);
+    --light-red: rgb(204, 124, 121);
+    --dark-yellow: rgb(104, 89, 2);
+    --light-yellow: rgb(194, 173, 53);
+    --dark-gray: #505050;
+    --light-gray: #b3b3b3;
+  }
+
+html {
+    font-size: 18px;
+    font-family: "Lato", sans-serif;
+}
+
+body {
+    margin: 0px;
+    background: var(--background-blue)
+}
+
+.header {
+    background-color: white;
+    padding: 5px 5px 5px 5px;
+    position: fixed;
+    width: 100%;
+    box-shadow: 3px 3px 3px var(--shadow-blue);
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: top;
+    top: 0px;
+}
+
+.main {
+    width: 80%;
+    display: flex;
+    flex-direction: column;
+    justify-content: top;
+    align-items: center;
+    font-size: 14px;
+    background: transparent;
+    margin-top: 120px;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.logo {
+    color: var(--logo-blue);
+    padding: 5px;
+    font-family: 'Comforter', cursive;
+    font-size: 42px;
+}
+
+.material-icons.md-18 { font-size: 18px; }
+.material-icons.md-24 { font-size: 24px; }
```

Comment:
unused css rules, if you copy-pasted it from external libraries consider extracting into a new file (`vendors.css`/`md.css`/whatever)